addons/sale_timesheet/models/project_overview.py

```diff
@@ -58,17 +61,25 @@ def _plan_prepare_values(self):
         # hours from non-invoiced timesheets that are linked to canceled so
         canceled_hours_domain = [('project_id', 'in', self.ids), ('timesheet_invoice_type', '!=', False), ('so_line.state', '=', 'cancel')]
         total_canceled_hours = sum(self.env['account.analytic.line'].search(canceled_hours_domain).mapped('unit_amount'))
-        dashboard_values['hours']['canceled'] = float_round(total_canceled_hours, precision_rounding=hour_rounding)
-        dashboard_values['hours']['total'] += float_round(total_canceled_hours, precision_rounding=hour_rounding)
+        canceled_hours = float_round(total_canceled_hours, precision_rounding=hour_rounding)
+        if is_uom_day:
+            # convert time from hours to days
+            canceled_hours = round(uom_hour._compute_quantity(canceled_hours, company_uom, raise_if_failure=False), 2)
```

Comment:
I think you should not add you own rounding since it's already configured on the uom and handled by `_compute_quantity` (everywhere you do that)

Reply:
Looks very much like a [floating point issue](https://docs.python.org/3/tutorial/floatingpoint.html) 
Maybe it's `_compute_quantity` which needs to be fixed ?

As a example of a floating point issue, try this in a python console:
```python
>>> 1.2 - 1.0
0.19999999999999996
```

Reply:
The issue comes from here: https://github.com/odoo/odoo/blob/master/odoo/tools/float_utils.py#L99
`_compute_quantity` uses `tools.float_round()` to round its value.
It is linked to the way the rounding is done: NORMALIZE -> ROUND -> DENORMALIZE
It happens during the denormalization.

One the following case happens for me:
```
value = 1.375
rounding_factor = 0.01

normalized_value = 137.5
rounded_value = 138.0

denormalized_value = rounded_value * rounding_factor => 1.3800000000000001
```
What is the best option ?
Do I change the way  `_compute_quantity` rounds its value by not using `tools.float_round()` ?

Reply:
Well, `float_round` seems to minimize floating point issue already. Could be even worse without using it :/ @tivisse what do you think ?

Reply:
It's correct IMHO.

The complete value is stored on the db, and displayed correctly by the web client and the decimal precision.

When displaying values "by hand", the float value have to be rounded manually.